### ISSUE TITLE
Wallaby falls on tsx test file changes.

### ISSUE THREAD

**Issue by nikolaymatrosov** [wallabyjs/public#333]
It works ok in Webstrom 11, but in VS2013 fails on any change in `SplashScreen-test.tsx`. I've created demo repo https://github.com/nikolaymatrosov/tsx-wallaby-vs2013

wallaby.js core v1.0.142 in both IDEs.

**Comment by ArtemGovorov** [comment]
What error does it display in VS for you?

**Comment by nikolaymatrosov** [comment]
Sometimes Wallaby can run tests properly and fails on test file change. Error message in Failig tests: `​​​​​Postprocessor run failure: undefined is not a function`.
Log messages from Wallaby - Console
```
wallaby.js started
core v1.0.142
Thu, 12 Nov 2015 13:15:24 GMT wallaby:project File cache: C:\Users\n.matrosov\AppData\Local\Temp\.wallaby\projects\Lzq3WkyJEWOj+apQfnttOJbjc=\cache
Thu, 12 Nov 2015 13:15:24 GMT wallaby:project package.json file change detected, invalidating local cache
Thu, 12 Nov 2015 13:15:24 GMT wallaby:workers Parallelism for initial run: 1, for regular run: 1
Thu, 12 Nov 2015 13:15:24 GMT wallaby:workers Starting run worker instance #0
Thu, 12 Nov 2015 13:15:24 GMT wallaby:workers Web server is listening at 63182
Thu, 12 Nov 2015 13:15:24 GMT wallaby:project File cache requires some updates, waiting required files from IDE
Thu, 12 Nov 2015 13:15:25 GMT wallaby:project File src/SplashScreen-test.tsx changes are patches only: false
Thu, 12 Nov 2015 13:15:25 GMT wallaby:project File src/SplashScreen.tsx changes are patches only: false
Thu, 12 Nov 2015 13:15:25 GMT wallaby:project Preparing to process src/SplashScreen-test.tsx
Thu, 12 Nov 2015 13:15:25 GMT wallaby:project Starting processor pool (if not yet started)
Thu, 12 Nov 2015 13:15:25 GMT wallaby:project File src/SplashScreen-test.tsx is not instrumented by core
Thu, 12 Nov 2015 13:15:25 GMT wallaby:project Preparing to process src/SplashScreen.tsx
Thu, 12 Nov 2015 13:15:25 GMT wallaby:project Starting processor pool (if not yet started)
Thu, 12 Nov 2015 13:15:25 GMT wallaby:project File src/SplashScreen.tsx is not instrumented by core
Thu, 12 Nov 2015 13:15:27 GMT wallaby:workers Started run worker instance (immediate) #0
Thu, 12 Nov 2015 13:15:28 GMT wallaby:processor No preprocessors found for src/SplashScreen-test.tsx
Thu, 12 Nov 2015 13:15:28 GMT wallaby:project Writing to disk and caching processed file src/SplashScreen-test.tsx
Thu, 12 Nov 2015 13:15:28 GMT wallaby:project File src/SplashScreen-test.tsx changes saved, store ts: 635829257154607400, change ts: 1447334125317
Thu, 12 Nov 2015 13:15:28 GMT wallaby:processor No preprocessors found for src/SplashScreen.tsx
Thu, 12 Nov 2015 13:15:28 GMT wallaby:project Writing to disk and caching processed file src/SplashScreen.tsx
Thu, 12 Nov 2015 13:15:28 GMT wallaby:project File src/SplashScreen.tsx changes saved, store ts: 635828138969904100, change ts: 1447334125336
Thu, 12 Nov 2015 13:15:28 GMT wallaby:project Stopping process pool
Thu, 12 Nov 2015 13:15:28 GMT wallaby:project Running postprocessor
Thu, 12 Nov 2015 13:15:28 GMT wallaby:postprocessor New TypeScript language service is required
Thu, 12 Nov 2015 13:15:28 GMT wallaby:project TypeError: undefined is not a function
  at StringScriptSnapshot.getText (C:\Projects\github\tsx-wallaby\node_modules\typescript\lib\typescript.js:41880:34)
  at createLanguageServiceSourceFile (C:\Projects\github\tsx-wallaby\node_modules\typescript\lib\typescript.js:42986:35)
  at acquireOrUpdateDocument (C:\Projects\github\tsx-wallaby\node_modules\typescript\lib\typescript.js:43099:34)
  at Object.acquireDocument (C:\Projects\github\tsx-wallaby\node_modules\typescript\lib\typescript.js:43088:20)
  at Object.getOrCreateSourceFile [as getSourceFile] (C:\Projects\github\tsx-wallaby\node_modules\typescript\lib\typescript.js:43713:41)
  at findSourceFile (C:\Projects\github\tsx-wallaby\node_modules\typescript\lib\typescript.js:35806:33)
  at processSourceFile (C:\Projects\github\tsx-wallaby\node_modules\typescript\lib\typescript.js:35760:27)
  at processRootFile (C:\Projects\github\tsx-wallaby\node_modules\typescript\lib\typescript.js:35699:13)
  at C:\Projects\github\tsx-wallaby\node_modules\typescript\lib\typescript.js:35445:60
  at Object.forEach (C:\Projects\github\tsx-wallaby\node_modules\typescript\lib\typescript.js:880:30)
  at Object.createProgram (C:\Projects\github\tsx-wallaby\node_modules\typescript\lib\typescript.js:35445:16)
  at synchronizeHostData (C:\Projects\github\tsx-wallaby\node_modules\typescript\lib\typescript.js:43651:33)
  at Object.getEmitOutput (C:\Projects\github\tsx-wallaby\node_modules\typescript\lib\typescript.js:46679:13)
  at C:\Users\n.matrosov\.wallaby\core\server.js:18:341
  at baseFor (C:\Users\n.matrosov\.wallaby\core\node_modules\lodash\index.js:1971:13)
  at baseForOwn (C:\Users\n.matrosov\.wallaby\core\node_modules\lodash\index.js:2025:14)
  at baseEach (C:\Users\n.matrosov\.wallaby\core\node_modules\lodash\index.js:1814:16)
  at Function.forEach (C:\Users\n.matrosov\.wallaby\core\node_modules\lodash\index.js:5701:11)
  at C:\Users\n.matrosov\.wallaby\core\server.js:18:147
  at Object.postprocessor (C:\Users\n.matrosov\.wallaby\core\server.js:15:29769)
  at C:\Users\n.matrosov\.wallaby\core\server.js:16:11164
  at [object Object]._onTimeout (C:\Users\n.matrosov\.wallaby\core\server.js:16:10766)
  at Timer.listOnTimeout (timers.js:89:15)
From previous event:
  at [object Object]._onTimeout (C:\Users\n.matrosov\.wallaby\core\server.js:16:10766)
  at Timer.listOnTimeout (timers.js:89:15)
Thu, 12 Nov 2015 13:15:28 GMT wallaby:project Test run finished
Thu, 12 Nov 2015 13:15:28 GMT wallaby:project Test run data re-queued
```

**Comment by ArtemGovorov** [comment]
Confirmed, we are working on the issue.

Not related to the issue, but I'm also getting the issue with missing `Function.bind` in your repo (`TypeError: 'undefined' is not a function (evaluating 'RegExp.prototype.test.bind(/^(data|aria)-[a-z_][a-z\d_.-]*$/)')`). If hit the same issue (and didn't find the solution yet) - [here is the explanation and the polyfill](https://github.com/wallabyjs/public/issues/328#issuecomment-155785081) you may use.

**Comment by ArtemGovorov** [comment]
Ah, never mind regarding the `Function.bind`, noticed that you are using PhantomJs 2.

**Comment by nikolaymatrosov** [comment]
That's the reason why I switched to PhantomJs 2. In the road map there is said that you planned to add support of other browsers. Does this feature have high priority?

**Comment by ArtemGovorov** [comment]
Not the highest at the very moment, but it's still at the top of the list for us.

**Comment by ArtemGovorov** [comment]
The issue is fixed in VS extension v1.0.18.
